docs/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1 @@
+* xref:showcase.adoc[Showcase]
```

Comment:
It's better to leave the text blank here and either allow Antora to use the doctitle or set the navtitle attribute in the document header.